opentech/apply/funds/models/reviewer_role.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+
+
+class ReviewerRole(models.Model):
+    name = models.CharField(max_length=128)
+    icon = models.ForeignKey(
```

Comment:
Can you add a panel configuration here so that we can get an image chooser panel for the imges

https://docs.wagtail.io/en/latest/reference/contrib/modeladmin/create_edit_delete_views.html#changing-which-fields-appear-in-createview-editview

Reply:
Yes, I have done this on the tweaks branch. I didn't realize you could use the wagtail panels on taxonomy items, but of course you can.